HISTORY.rst

```diff
@@ -5,7 +5,7 @@ History
 ##### (##########)
 ------------------
 
-* Fix an unfixed random state in one of the classification tests
+* Add LTT for multilabel-classification
```

Comment:
ERROR: avoid deleting existing lines (unless it's a merge conflict, which you'll just have to take this comment into account when you do).
```suggestion
* Add Learn-Then-Test method for multilabel-classification
* Fix an unfixed random state in one of the classification tests
```